crates/bin/node/src/initialization/nuts_settings.rs

```diff
@@ -25,5 +25,9 @@ pub(super) fn nuts_settings() -> NutsSettings<Method, Unit> {
             }],
             disabled: false,
         },
+        nut19: nuts::nut19::Settings {
+            ttl: None,
+            cached_endpoints: vec![],
```

Comment:
yeah, we can probably remove this `cached_endpoints` config. And always cache mint quote, mint and melt